core/modules/file/file.module

```diff
@@ -3356,16 +3356,6 @@ function file_build_content($file, $view_mode = 'full', $langcode = NULL) {
   // Build the actual file display.
   // @todo Figure out how to clean this crap up.
```

Comment:
I think, this comment can go as well - we "figured out". :wink:

Reply:
@indigoxela done